src-v2/NTS.Persistance/Adapters/RankingRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿using NTS.Domain.Core.Entities;
+using NTS.Persistence.States;
+
+namespace NTS.Persistence.Adapters;
+
+public class RankingRepository(IStore<CoreState> store) : SetRepository<Ranking, CoreState>(store)
+{
+}
```

Comment:
As it is currently there are two `IRepository<Ranking>` implementations. Rename ClassificationRepository instead of creating a new RankingRepository.